packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -87,7 +87,11 @@ export const orchestratorApi = createApi({
                     prepareHeaders,
                     customErrors: (error) => {
                         const { name, message, stack, response } = error;
-                        if (response?.errors && response.errors?.length > 0) {
+                        if (
+                            response?.errors &&
```

Comment:
`response?.errors && response.errors?.length > 0` can be replaced with `response?.errors?.length > 0` for better readability.

Reply:
you don't even need the `> 0`

Reply:
Can you do a 'code suggestion' for these types of comments next time?